doc/src/explore/wallet-browser.md

```diff
@@ -132,6 +130,23 @@ You can send coins from your wallet to another address.
 1. In the **Amount** field, enter the number of SUI to send, and then click **Continue**.
 1. Enter the recipient's address, then click **Send Coins Now**.
 
+## Stake and earn SUI (Testnet only)
+
+While the Testnet network is available, you can try out staking to earn SUI. When you stake SUI, you delegate your SUI tokens to a validator to stake. The validator then pays you rewards for delegating your SUI to stake. Note that SUI tokens have no value. The rewards are for testing purposes and have no real value.
+
+1. Open your wallet and click **Coins**.
+1. Click **Stake & Earn SUI**.
+1. Select a validator to stake with.
+1. Choose an amount of SUI to stake.
+   Be sure to enter an amount that leaves enough SUI in your wallet to cover gas fees.
+1. Click **Stake Now**.
+
+Your stake starts earning rewards at the start of the next epoch.
+
+## View current stake
+
+To view details about your current stakes, click Currently Staked on the Coins tab of the Wallet. Details include: the amount you staked, the validator you chose, amount earned, and the validator commission.
```

Comment:
```suggestion
To view details about your current stakes, click **Currently Staked** on the **Coins** tab of the Wallet. Details include: the amount you staked, the validator you chose, amount earned, and the validator commission.
```

Reply:
sorry I added that after I created the PR, thanks for catching